app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -16,12 +16,15 @@ import Competition from './SubForms/Competition';
 
 const CONTACT_RECIPIENTS = [
   'competition',
-  'communications_team',
-  'results_team',
-  'software',
+  'communicationsTeam',
+  'resultsTeam',
+  'softwareTeam',
 ];
 
-const CONTACT_RECIPIENTS_MAP = _.keyBy(CONTACT_RECIPIENTS);
+const CONTACT_RECIPIENTS_MAP = CONTACT_RECIPIENTS.reduce((acc, recipient) => {
+  acc[recipient] = _.snakeCase(recipient);
+  return acc;
+}, {});
```

Comment:
```suggestion
const CONTACT_RECIPIENTS_MAP = _.keyBy(CONTACT_RECIPIENTS, _.camelCase);
```

Reply:
**NOTE** This only works if you undo the diff above and keep `CONTACT_RECIPIENTS` in snake_case.
Lodash only has a method for turning array into object when you create keys of the object and retain the values. There is no equivalent shorthand method for iterating over an arrays as keys and determining values.

Reply:
Done, thanks